devops/arangodb.yml

```diff
@@ -46,6 +52,10 @@
       shell: ARANGODB_DEFAULT_ROOT_PASSWORD={{ arangodb_root_password }} arango-secure-installation
       become: true
 
+    - name: Ensure there is a read-only user
+      command: arangosh --server.password {{ arangodb_root_password }} --server.endpoint ssl://127.0.0.1:8529 --javascript.execute-string "const users = require('@arangodb/users'); if (!users.exists('{{ arango_readonly_username }}')) { users.save('{{ arango_readonly_username }}', '{{ arango_readonly_password }}'); } users.grantDatabase('{{ arango_readonly_username }}', '*', 'ro');"
```

Comment:
For readability, can you pull the script into a file, and use the appropriate `arangosh` flag to read from that instead? If that simple transformation doesn't work because of variable interpolation, there is also the [template](https://docs.ansible.com/ansible/latest/modules/template_module.html) module.

Reply:
Ahh, I wasn't aware of the template module. I'll try this out.